config.py

```diff
@@ -2,8 +2,16 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 # -- Alarm Settings
-PIN = 10                       # Pin to which the PowerSwitch Tail is connected
-ALARM_DURATION = 15            # Light duration in minutes
+# Alarm will sound for alarm duration or PIR sense whichever occures first.
+
+# -- Pin Configuration as per BCM Model
+PIN = 15                      # Pin to which the PowerSwitch Tail is connected
+ALARM_DURATION = 0.1            # Alarm Sound duration in minutes
+#LIGHT_DURATION = 10			  # Light switch duration in minutes.
+RADIO_DURATION = 15			  # Radio duration in minutes
+PIR = 4 					  # Pin to which PIR sensor or Switch is connected
```

Comment:
Looks like there are mixed tabs and spaces; I think we should use only spaces, as per the [PEP8 Specification](https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces)